apps/client/src/app/pages/resources/glossary/resources-glossary.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'gf-resources-glossary',
+  templateUrl: './resources-glossary.component.html',
```

Comment:
Move down to order alphabetically

Reply:
this is what you meant right?
![image](https://github.com/user-attachments/assets/1f82105f-931a-4d8d-9258-987141273215)

Reply:
> this is what you meant right?

Yes 👍🏻